libreport.spec

```diff
@@ -286,8 +287,6 @@ data over ftp/scp...
 %autosetup
 
 %build
-sed 's|DEF_VER=.*$$|DEF_VER='%{version}'|' -i gen-version
-#./gen-version
 ./autogen.sh
 autoconf
```

Comment:
https://github.com/abrt/libreport/pull/684

This should be applied to the "new" spec.

Reply:
but the spec.in is the old and the spec is the new one.